bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -169,7 +171,6 @@ private func dummy() {
     _ = NSLocalizedString("How to edit your page", comment: "")
     _ = NSLocalizedString("How to edit your post", comment: "")
     _ = NSLocalizedString("Image caption. %s", comment: "translators: accessibility text. %s: image caption.")
-    _ = NSLocalizedString("Image settings", comment: "")
```

Comment:
- Removal
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/image/image.js#L334:L334)

This string is already translated as it's used in the web version, so we don't need to include it here.